packages/demo/src/plasma/routes/input/pages/MultiSelectExamples.tsx

```diff
@@ -0,0 +1,326 @@
+    {displayValue: 'Disabled', value: 'disabled', disabled: true},
+    {displayValue: 'Three', value: '3'},
+    {displayValue: 'Four', value: '4'},
+    {
+        displayValue: 'Five',
+        value: '5',
+        selectedTooltip: {
+            title: 'HighFive!',
+            placement: TooltipPlacement.Bottom,
+        },
+    },
+    {displayValue: 'Six', value: '6'},
+    {displayValue: 'Seven', value: '7', selectedDisplayValue: 'James Bond 007'},
+];
+
+const defaultFlatSelectOptions: IFlatSelectOptionProps[] = [
+    {id: UUID.generate(), option: {content: 'All'}, selected: true},
+    {id: UUID.generate(), option: {content: 'even'}},
+    {id: UUID.generate(), option: {content: 'odd'}},
+];
+
+const WithNonEmptyMultiSelect = withNonEmptyMultiSelectHOC(MultiSelectConnected);
+
+const WithNonEmptyMultiSelectExample = () => (
+    <LabeledInput
+        label="A Multi Select with Non Empty Validation"
+        message={
+            <div>
+                <ValidationMessage id="multi-select-non-empty" />
+            </div>
+        }
+    >
+        <br />
+        <WithNonEmptyMultiSelect id="multi-select-non-empty" items={defaultItems} />
+    </LabeledInput>
+);
+
+export interface IMultiSelectExamplesState {
+    first: IItemBoxProps[];
+    drag: IItemBoxProps[];
+    second: IItemBoxProps[];
+    hoc: IItemBoxProps[];
+    markdown?: string;
+}
+
+const MultiSelectWithInitialValues = withInitialValuesMultiSelectHOC(MultiSelectConnected);
+const MultiSelectWithInitialValuesExample = () => (
+    <Section level={3}>
+        <LabeledInput
+            label="A Multi Select with initial values"
+            message={
+                <div>
+                    <ValidationMessage id="multi-select-initial-values" />
+                </div>
+            }
+        >
+            <MultiSelectWithInitialValues
+                initialValues={[defaultItems[0].value, 'INVALID ONE']}
+                id="multi-select-initial-values"
+                items={defaultItems}
+            />
+        </LabeledInput>
+    </Section>
+);
+
+export class MultiSelectExamples extends React.Component<null, IMultiSelectExamplesState> {
+    static description =
+        'A multi-select component allows users to select one or more options from a set of predefined options, or, if relevant, to add custom options.';
+
+    constructor({props, state}: {props: null; state: IMultiSelectExamplesState}) {
+        super(props, state);
+
+        const second = _.map(defaultItems, (item) => _.clone(item));
+        second[0].selected = true;
+
+        const hoc = _.map(defaultItems, (item) =>
+            _.extend({}, item, {append: {content: () => <span className="append ml1">{item.value}</span>}})
+        );
+        hoc[0].selected = true;
+        hoc[1].selected = true;
+
+        this.state = {
+            first: _.clone(defaultItems),
+            drag: _.clone(defaultItems),
+            second,
+            hoc,
+            markdown: '',
+        };
```

Comment:
You can use Vanilla JS instead of underscore for this part

Reply:
This is out of scope for this pr I think, I know Jack just pushed the changes but originally, he just moved around the files, I would put this changes in another pr just to make sure that it can be tested on the side, not in this big pr.

Reply:
I removed underscore anyway, didn't take long but I will avoid future changes like this in this PR